### DISCUSSION TITLE
How to set the default <body> background colour

### DISCUSSION THREAD

**Discussion by Indy-Dev** [chakra-ui/chakra-ui#2207] [Q&A] [answered]
Hi all,

Our project will _not_ be implementing a dark/light color mode switcher for now, but I'd like the set the background color of the `<body>` tag. How is that accomplished please as I cannot find anything in the docs?

I'd also like the set other properties on the `<html>` and `<body>` tags e.g. `height: 100%`

Many thanks

**Comment by Indy-Dev** [comment]
I've accomplished it with

```
import { Global } from '@emotion/core';

<Global styles={
  {
    backgroundColor: '#212121',
  }
} />
```

Is this the best approach?

**Comment by with-heart** [comment, marked as answer]
Hi @Indy-Dev, you can use `theme.styles.global` for that! You can find documentation on that here: https://next.chakra-ui.com/docs/features/global-styles

**Reply by Indy-Dev** [reply]
That's great. Didn't realist there was a version (V.1.X) specifically for NextJJ.

Just upgraded and gonna try your solution.

Thanks for your help.

Library is very nice by the way.

**Reply by with-heart** [reply]
Thank you! ❤️
